roles/haproxy/templates/etc/haproxy/haproxy_openstack.cfg

```diff
@@ -116,7 +116,7 @@
 
 global
   log /dev/log local0
-  maxconn 256
+  maxconn 4096
```

Comment:
If the larger maxconn is only needed by mysql cluster,  this option may be under the mysql configure section.